blocks/file-blocks/excalidraw/index.tsx

```diff
@@ -16,10 +16,17 @@ if (typeof window !== "undefined") {
 export default function (props: FileBlockProps) {
   const { context, content, isEditable, onUpdateContent } = props;
   const [excalModule, setExcalModule] = useState<any>(null);
+  const [version, setVersion] = useState<number | null>(null);
```

Comment:
you could use a ref here instead of state, you don't need to re-render when the version is set

Reply:
Good call! Will update